firmware/controllers/engine_cycle/knock_controller.cpp

```diff
@@ -8,10 +8,14 @@
 #include "pch.h"
 #include "knock_logic.h"
 
+void KnockController::init(engine_configuration_s const * const previousConfig) {
+	m_maxRetardTable.init(config->maxKnockRetardTable, config->maxKnockRetardLoadBins, config->maxKnockRetardRpmBins);
```

Comment:
note this flips the load/rpm bins to match the current implementation

Reply:
not the minimal diff to fix the bug, sigh